src/components/Cards.js

```diff
@@ -91,6 +27,10 @@ const CardItem = styled.li`
    list-style-type: none;
    width: 30%;
    margin-bottom: 10px;
+   margin-right: 15px;
+   :nth-child(3n) {
```

Comment:
Can you confirm that this works? If not you might have to write it as `&:nth-child`?

Reply:
This does work as is.